dynamic_models/models.py

```diff
@@ -115,6 +116,7 @@ class FieldSchema(models.Model):
     model_schema = models.ForeignKey(ModelSchema, on_delete=models.CASCADE, related_name="fields")
     class_name = models.TextField()
     kwargs = FieldKwargsJSON(default=dict)
+    db_name = models.CharField(max_length=63, null=True)
```

Comment:
I don't believe this is needed since each model should only have one DB. It wouldn't make sense if the model DB was a different value than the field DB, or if multiple fields had different DB values.

We should be able to remove this field and get the DB name from the model schema instead.